backend/substrapp/tasks/tasks.py

```diff
@@ -473,6 +486,8 @@ def prepare_materials(subtuple, tuple_type):
         put_composite_models(subtuple, subtuple_directory, models_content)
     elif tuple_type == 'traintuple' and models_content:
         put_models(subtuple, subtuple_directory, models_content)
+    elif tuple_type == 'aggregatetuple' and models_content:
+        put_models(subtuple, subtuple_directory, models_content, model_hash)
```

Comment:
`model_hash` should not be used for calling `put_models`?

Reply:
Good catch!